docs/products/flink/howto/ext-pg-cdc-jdbc-connector.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Create a PostgreSQL® CDC or JDBC connector-based Apache Flink®
+---
+
+Learn how to create a table in your Apache Flink® application to interact with an external PostgreSQL® database.
+
+You can choose between two methods:
+
+- **Change Data Capture (CDC)**: Process real-time changes from the PostgreSQL tables
+  using Debezium.
+- **JDBC**: Execute SQL queries directly on the PostgreSQL database.
+
+## Prerequisites
+
+- An active Aiven for Apache Flink service.
+- An external PostgreSQL database with access details.
+- [Integration](/docs/products/flink/howto/ext-pg-flink-integration) between
+  Aiven for Apache Flink and external PostgreSQL.
+
+## Configure external PostgreSQL connector
+
+Follow these steps to configure an external PostgreSQL connector in your Aiven for Apache Flink
+application table, using either CDC or JDBC:
+
+1. Go to the **Aiven for Apache Flink** service page and
+   click **Application** in the sidebar.
+1. Click **Create new application**, enter a name your application, and
+   click **Create application**.
+
+   :::note
+    If modifying an existing application, create a new version for source or s
+    ink table changes.
+   :::
+
+1. Click **Create first version** to create the first version of your application.
+1. Click **Add your first source table** to begin setting up your data source.
+1. In the **Add new source table** screen, select the *external PostgreSQL* as the
+   integrated service.
+1. In the **Table SQL**, enter the SQL statement for your chosen connector:
+
+   - For a **CDC connector**:
+
+     ```sql
+      CREATE TABLE external_pg_cdc_source (
+        id INT PRIMARY KEY,
+        data VARCHAR
+      ) WITH (
```

Comment:
From my experimenting, it was necessary to add a `slot.name` parameter as well, e.g.:
`'slot.name' = 'my_slot_name'`

Reply:
Thanks. Added this.